AngularSpecificAttributesRemover.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const jestDOMElementSerializer = require('pretty-format').plugins.DOMElement;
+
+const serialize = (node, ...rest) => {
+  const nodeCopy = node.cloneNode(true);
+  Object.keys(nodeCopy.attributes)
+    .map(key => nodeCopy.attributes[key].name)
+    .filter(attributeName => attributeName.includes('ng-reflect'))
+    .forEach(attributeNameToRemove => nodeCopy.attributes.removeNamedItem(attributeNameToRemove));
+
+  return jestDOMElementSerializer.serialize(nodeCopy, ...rest);
+};
+
+const test = val =>
+  isElement(val) && jestDOMElementSerializer.test(val) && serializeTestFn(val);
```

Comment:
Is `isElement` really required or should the test function from `DOMElement` not be enough already?

Reply:
Hm... I think I got some problem yesterday without it.... but now everything works okay... Looks like it is not needed. I removed it.

Reply:
It was for case when val.attributes is undefined.

I added check for it and now everything is okay.